### DISCUSSION TITLE
No Change After Publishing these days

### DISCUSSION THREAD

**Discussion by dannyleungkc** [swellstores/community#134] [Help]
Hi there,

These days when I publish changes of the storefront. It's not updated after publishing. I tried to delete cache data in my browsers but still the same. Does anyone experience this issue too?

Every time I hit publish, it's rather quick to finish.

Regards,
Danny

**Comment by sags95** [comment]
@dannyleungkc Are you seeing any errors in the deployment logs? You can access these by clicking the button to the left of 'Preview' when accessing the editor.  If you select the **Production** radio button there might be something there causing an error.


<img width="1332" alt="Screenshot 2023-02-14 at 12 50 45 PM" src="https://user-images.githubusercontent.com/9522171/218839964-38df2a5c-2d67-4afa-a66d-9c7c4c7d9e0e.png">

**Reply by dannyleungkc** [reply]
mmm... My interface is different from yours as I don't have the rolling paper icon button.

**Reply by dannyleungkc** [reply]
Sorry but my interface is just simply different from yours.

**Reply by sags95** [reply]
@dannyleungkc Would you be able to reach out to support@swell.is and mention the store id that is experiencing this issue?

**Comment by Strongeyed** [comment]
Hi Danny, I'm experiencing exactly the same issue. Publishing goes from 'starting to publish' to completed within seconds and the changes are not reflected on the frontend of two of my stores. Here is a screenshot of one of the errors but I'm not sure if that relates to this issue. Did you manage to resolve this issue? Keen to hear what you ended up doing!

![Screenshot 2023-03-17 at 11 17 39 AM](https://user-images.githubusercontent.com/128098520/225764943-497c066a-ba62-4b8d-8e65-b265d50e715e.png)

**Comment by Strongeyed** [comment]
Any updates from anyone? Still having the same issues, how did people that reported this in January/February get around this? As for me it renders my store unusable as I can't update new stock. Keen to hear some thoughts!

**Comment by rodrigoddalmeida** [comment]
@dannyleungkc @Robin-DG  could you update your storefronts (Origin) to the latest version, then publish? That should take care of the error you see. You can do it through the Admin Dashboard UI or by rebasing/merging from Origin's public repository if you have custom code.

**Reply by Strongeyed** [reply]
@rodrigoddalmeida Absolutely awesome Rodrigo, this made my day to be honest. I'm still having quite a couple of publication errors that I have recorded in a video that I could send through in a PM?

**Reply by Strongeyed** [reply]
One of the issues is that the website is now displaying code in the buttons etc. as per the image below;

![IMG_6969](https://user-images.githubusercontent.com/128098520/226728598-a927a078-c54b-481e-849d-4ddb2b04a8cb.jpg)

**Reply by Strongeyed** [reply]
Still having issues with publishing now, my website only published once but won't do it again now

**Reply by dannyleungkc** [reply]
> Still having issues with publishing now, my website only published once but won't do it again now

Same here, I gave up Swell and moved to somewhere else. Very little technical support unfortunately...

**Comment by mewa** [comment]
Yes, we've also had issues where nothing happens after clicking "Publish". Although in our case no error logs are present - they're just too short, so you can see nothing has happened.

**Reply by Strongeyed** [reply]
@mewa Must be a global thing as I have 3 stores with that issue now, when did it start happening to you? For me it started 4 weeks ago.

**Comment by logeshswell** [comment]
@Strongeyed @mewa @dannyleungkc Please share your store URL and I can help you with it.
